### ISSUE TITLE
Project Management Status

### ISSUE THREAD

**Issue by mcdevitts** [scanny/python-pptx#870]
@scanny is this project dead? Are you intending to continue active management of this project? Is there a way we can improve issue resolution and PR resolution?

We use this project on a daily basis, and I would like to see it, at least, actively maintained for simple bug fixes and python version compatibility improvements. If this project is dead, perhaps the community should consider creating a semi-official `python-pptx2`?

**Comment by JimInCO** [comment]
Agreed, one PR that is of particular interest to me is #771. Is there anyway in which the active users of this package can help?

**Comment by niwreg-coder** [comment]
I am using this package now frequently, and would like to start using it even more. Therefore, it would be needed that the awesome work by Scanny for Python-docx and Python-pptx is not lost but continued (by the community). If there is anything I could do, please let me know

**Comment by M22Y** [comment]
Would help to know if the project is dead. I/we depend on this a lot on the daily. if possible for someone to pick up management on this repo or a fork.

**Comment by buzzm** [comment]
Steve…?   :-)

**Comment by Dasc3er** [comment]
Hi all, I just proposed the creation of a GitHub org for maintaining a fork of this project in #898 (some thoughts there).
What do you think about this?
